src/mono/wasm/wasm.proj

```diff
@@ -198,7 +198,8 @@
 
     <ItemGroup>
       <ICULibNativeFiles Include="$(ICULibDir)/libicuuc.a;
-                                  $(ICULibDir)/libicui18n.a" />
```

Comment:
Should we include `$(ICULibDir)/*.a` instead?

Reply:
I think being explicit is better here